### PR TITLE
expiration: Add a few metrics to measure revoke queue lengths

### PR DESCRIPTION
**Pull request by briankassouf**

### CODE REVIEW

**Review by swayne275 — APPROVED** [review]
Cool! I've got one coming up for per-queue length (got slowed down a bit trying to figure out how to test the metrics but maybe that isn't needed)

File: helper/fairshare/jobmanager.go [hunk 1]
```diff
@@ -110,6 +112,9 @@ func (j *JobManager) AddJob(job Job, queueID string) {
 	}
 
 	j.queues[queueID].PushBack(job)
+	j.totalJobs++
+	metrics.AddSampleWithLabels([]string{j.name, "job_manager", "queue_length"}, float32(j.queues[queueID].Len()), []metrics.Label{{"queue_id", queueID}})
+	metrics.SetGauge([]string{j.name, "job_manager", "total_jobs"}, float32(j.totalJobs))
```
Comment (mgritter) [inline helper/fairshare/jobmanager.go:117]: I think including the name first is undesirable.  job_manager.queue_length.{name} would be better, or use a label. 

Also, sorry to bring this up again, but I'd really like it if new metrics we add have cluster labels, as this activity can bounce around different hosts.  (I think that perf standbys are still going to just remove from cache directly, not use a worker pool?)

Comment (briankassouf) [reply helper/fairshare/jobmanager.go:117]: You're correct about standbys not using the worker pool, I'll switch this to adding the cluster label. I however think the name being first is preferable so we can keep all the expiration related metrics under the same prefix: `expire.*`.

Comment (mgritter) [reply helper/fairshare/jobmanager.go:117]: So, each instance would be documented separately in the telemetry guide?

Comment (mgritter) [reply helper/fairshare/jobmanager.go:117]: (And maybe we need some documentation, now that I think of it.)

**Review by swayne275 — APPROVED** [review]

**Pull request merged**